src/IOST/Account.cpp

```diff
@@ -0,0 +1,50 @@
+#include "Account.h"
+#include "../Base58.h"
+
+using namespace TW;
+using namespace TW::IOST;
+
+bool Account::isValid(const std::string& s) {
+    if (s.size() < 5 || s.size() > 11) { 
+        return false; 
+    }
+    for (char ch : s) {
+        if (!((ch >= 'a' && ch <= 'z') || (ch >= '0' && ch <= '9') || ch == '_')) {
```

Comment:
I find this easier to follow:
```suggestion
        if ((ch < 'a' || ch > 'z') && (ch < '0' || ch > '9') && ch != '_') {
```

Reply:
I think the original version is more friendly for readers?